admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -64,7 +290,7 @@ you can set the following parameters inside the :file:`config/config.php`.
 ::
 
   <?php
-  $CONFIG = array (
+   $CONFIG = array (
```

Comment:
Please revert the whitespace change

Reply:
Sorry still trying to get a handle on this sphynix thing